Storage.md

```diff
@@ -80,7 +79,7 @@ ItemsHolder.addItem("color", {
 ItemsHolder.setItem("color", "red");
 ```
 
-Modularity restricts an item's value to a range from 0 to the specified max and runs the `onModular` function as many times until the value hits the modular range.
+Modularity restricts an item's value to a range from 0 to the specified max and keeps running the `onModular` function until the value hits the modular range.
```

Comment:
This is a little vague: is it that it immediately runs `onModular`, or runs it when the item is changed, or...?